### PR TITLE
expose availableTechnologies

### PR DESCRIPTION
**Pull request by semafor**
Expose availableTechnologies [1] as a QStringList.

[1] http://git.kernel.org/cgit/network/ofono/ofono.git/tree/doc/radio-settings-api.txt#n48

### CODE REVIEW

File: src/qofonoradiosettings.h [hunk 1]
```diff
@@ -28,7 +28,8 @@ class QOFONOSHARED_EXPORT QOfonoRadioSettings : public QOfonoModemInterface
 {
     Q_OBJECT
     Q_PROPERTY(QString modemPath READ modemPath WRITE setModemPath NOTIFY modemPathChanged)
-    Q_PROPERTY(QString technologyPreference READ technologyPreference WRITE setTechnologyPreference NOTIFY technologyPreferenceChanged)
+    Q_PROPERTY(QString availableTechnologies READ availableTechnologies NOTIFY availableTechnologiesChanged)
+    Q_PROPERTY(QStringList technologyPreference READ technologyPreference WRITE setTechnologyPreference NOTIFY technologyPreferenceChanged)
```
Comment (lpotter) [inline src/qofonoradiosettings.h]: I think you have these two backwards. technologyPreference is a string and availableTechnologies should be a stringlist.

Comment (semafor) [inline src/qofonoradiosettings.h]: Fixed!

**Comment by monich** [comment]
Please squash these commits into one.

**Comment by semafor** [comment]
Done!

**Comment by monich** [comment]
LGTM

**Pull request merged**